docs/assets/images/push_job_steps.drawio

```diff
@@ -0,0 +1,250 @@
+<mxfile host="Electron" modified="2024-02-26T21:48:03.768Z" agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/23.1.5 Chrome/120.0.6099.109 Electron/28.1.0 Safari/537.36" etag="BhlpWibByn0qgMP8agoQ" version="23.1.5" type="device">
```

Comment:
Sorry that this is unclear. The "Server Read Traffic SERVING" step is the swap. From my understanding, if there are DVC instances reporting their status into the local DVPS system store, then the child controller will wait for all heartbeating DVC instances to report their completion before doing the swap, and therefore the server swap depends on the DVC swap. Please LMK if this is not the right understanding (or if you think we should change this behavior for some reason...).

And so regarding the swap being in the per-region flow, it is there, but named as "serving" rather than "swap".

Reply:
I added notes in the intro to capture the above details.